src/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Spaces } from '../..';
+import { Box, BoxProps } from '../Box/Box';
+
+const COLUMN_WIDTH = 100 / 12;
+const GAP = Spaces[3];
+
+const Row = styled(Box).attrs({ display: 'flex', flexDirection: 'row', flexWrap: 'wrap' })`
+    gap: ${GAP};
+    margin-right: -${GAP};
```

Comment:
I think we can find a way to avoid setting negative margin-right. This is not a `gap` issue, because it sent only spacing between components in a flexbox container.

I suspect this is the effect of our usage of `calc` which assume that each column has a gap. This assumption is incorrect, the amount of gaps is the amount of columns minus one. If we fix the formula, we fix the issue

Reply:
Makes sense. Let's check if we want to use grid or flex, and then decide if I should fix this.

Reply:
I don't have a strong opinion about it as long as we can use it to create a layout easily